packages/search/src/search.ts

```diff
@@ -64,13 +64,29 @@ export class Search extends Textfield {
         }
     }
 
-    public reset(): void {
+    public async reset(): Promise<void> {
         /* istanbul ignore if */
         if (!this.form) {
             return;
         }
         this.value = '';
         this.form.reset();
+        await this.updateComplete;
+        this.focusElement.dispatchEvent(
+            new InputEvent('input', {
+                bubbles: true,
+                composed: true,
+            })
+        );
+        // The native `change` event on an `input` element is not composed,
+        // so this synthetic replication of a `change` event must not either
```

Comment:
did you mean "must not *be* either"?

Reply:
English really is the hardest part of the work we do here... corrected!